Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -402,6 +403,23 @@ WarpXOpenPMDPlot::~WarpXOpenPMDPlot ()
   }
 }
 
+
+void WarpXOpenPMDPlot::flushCurrent(bool isBTD) const
+{
+     WARPX_PROFILE("WarpXOpenPMDPlot::flushCurrent");
+     // open files from all processors, in case some will not contribute below
```

Comment:
These comments can be adapted to write a doc-string for the function declaration